src/Network/Receive/ServerType0.pm

```diff
@@ -2875,11 +2875,13 @@ sub received_characters {
 		$chars[$slot]{int} = $int;
 		$chars[$slot]{dex} = $dex;
 		$chars[$slot]{luk} = $luk;
-		$chars[$slot]{sex} = $accountSex2;
+		$chars[$slot]{sex} = ($masterServer->{charBlockSize} == 145 && $masterServer->{serverType} =~ /^iRO/) && (unpack( 'C', substr($args->{RAW_MSG}, $i + $blockSize -1)) =~ /^0|1$/)? unpack( 'C', substr($args->{RAW_MSG}, $i + $blockSize -1)) : $accountSex2;
```

Comment:
Maybe we should make this a servers.txt option, so other servers with the same behavior can make use of it.

Reply:
If we know other servers send the char sex at last byte ... but i don't.
I don't play other servers , and i have very limited bandwith to try other servers , so i limit to iRO only